README.md

```diff
@@ -1,3 +1,4 @@
+[![Build Status](https://travis-ci.org/andela/ah-backend-prime.svg?branch=develop)](https://travis-ci.org/andela/ah-backend-prime)
```

Comment:
@anyatibrian why does the travis badge indicate a failing build.

Reply:
its because my branch has not yet been merged to develop